pkg/gcloud/gcloud.go

```diff
@@ -61,14 +62,22 @@ func (g *gCloudSvc) Instances(serversMap map[string]server.Server) error {
 		log.Debug(len(instances))
 
 		for _, instance := range instances {
-			newServer := server.Server{}
-			newServer.Tags = make(map[string]string)
-			newServer.Name = instance.Name
-			newServer.Address = instance.NetworkInterfaces[0].AccessConfigs[0].NatIP
-			for index, key := range instance.Tags.Items {
-				newServer.Tags[strconv.FormatInt(int64(index), 10)] = key
+			if len(instance.NetworkInterfaces) > 0 {
+				newServer := server.Server{}
+				newServer.Tags = make(map[string]string)
+				newServer.Name = instance.Name
+				newServer.Address = instance.NetworkInterfaces[0].AccessConfigs[0].NatIP
```

Comment:
What happens if AccessConfigs is an empty slice?

Reply:
Would love to pair on this as I've tried a few different ways to approach this, however, the only way I know to address the conditional statement is to check if newServer.Address has any characters/strings associated to it (>0), if you have any ideas, I'd love to work with you on that!!